### DISCUSSION TITLE
Feature Request - Enhancing User Experience by Rethinking Bookmark Button Placement

### DISCUSSION THREAD

**Discussion by iammrwaffleman** [dailydotdev/daily#925] [Feature Requests]
Dear Daily.dev Team,

I hope this message finds you well. As an avid user of your platform, I have enjoyed the wealth of knowledge and insights daily.dev offers. Today, I am writing to share a suggestion that could potentially improve the user experience on your platform.

Currently, the bookmark button is located within the share menu and the 3-dot menu at the top right of each post thumbnail. While this setup is functional, it sometimes creates confusion and additional steps for users who wish to quickly save articles for later. Notably, the association of bookmarking—an inherently personal action—with sharing—an action directed towards others—doesn't seem quite justifiable from a user experience standpoint.

In light of this, I propose moving the bookmark button to a more prominent and easily accessible location—specifically, the top right corner of each post thumbnail. This adjustment would make the bookmarking process more immediate, intuitive, and less disruptive to the user's navigation.

With this change, users could bookmark articles more efficiently, which could in turn lead to increased engagement as users might find it easier to save (and consequently, read) more articles.

I understand that decisions regarding design and features involve careful consideration of many factors and stakeholder inputs. However, I believe this relocation of the bookmark button would have a significant positive impact on the overall user experience.

Thank you for your attention to this matter. I have faith in daily.dev's commitment to continually enhance the platform based on user feedback.

Looking forward to your positive consideration and action.

Best Regards,
mr.waffleman

**Comment by idoshamun** [comment]
Hi @iammrwaffleman,

Thank you sincerely for your thorough explanation.
We'll consider it and decide how to proceed. 🙏

**Comment by nimrodkra** [comment]
@iammrwaffleman done. It’s back on the card itself - enjoy and thank for the feedback!

![IMG_8444](https://github.com/dailydotdev/daily/assets/41470823/fedfaf5a-1248-4781-b8e4-cc1d8ea8f020)
